tests/dummy/app/pods/toggle/controller.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember'
+const {Controller} = Ember
+
+export default Controller.extend({
+
+  isButtonToggled: false,
```

Comment:
Also if there is a value mapping present, does this work?  E.g. if the onValue is set to 'foo' and I set toggled='foo' will the toggle button be `on`?